source/features/split-issue-pr-search-results.tsx

```diff
@@ -12,20 +12,51 @@ function cleanLinks(): void {
 	}
 }
 
+function getPageSearchQuery(): SearchQuery {
+	const searchParameters = new URLSearchParams(location.search);
+	return new SearchQuery(searchParameters);
+}
+
 type GitHubConversationType = 'pr' | 'issue';
 
 function createUrl(type: GitHubConversationType, pathname = location.pathname): string {
 	const url = new URL(pathname, location.origin);
-	url.searchParams.set('q', pageSearchQuery.get() + ` is:${type}`);
+	const searchQuery = getPageSearchQuery();
+	// Remove all existing is:pr/issue query parts, only the last is used anyways
+	searchQuery.remove('is:pr', 'is:issue');
+	searchQuery.add(`is:${type}`);
```

Comment:
There’s a replace method that accepts a regex

Reply:
Good idea, didn't think about using a regex 😅